src/encoded/static/components/award.js

```diff
@@ -1989,6 +2010,23 @@ class CumulativeGraph extends React.Component {
         require.ensure(['chart.js'], (require) => {
             const Chart = require('chart.js');
             const ctx = document.getElementById('myGraph').getContext('2d');
+            const xticksBallParkCount = 25; // ballpark number desired x-axis ticks
+            const monthReleasedLength = monthReleased.length;
+            const lastxAxisIndex = monthReleasedLength - 1;
+            const xticks = monthReleasedLength < xticksBallParkCount ? 1 : Math.ceil(monthReleasedLength / xticksBallParkCount);
+            const dateValues = [];
+            const totals = [];
+
+            // eslint-disable-next-line no-plusplus
```

Comment:
I’d like to not have any new exceptions to ESLint rules, and gradually remove the exceptions we have from code written before we used ESLint. This one seems like an easy rule to follow without disabling it.

Reply:
@forresttanaka Corrected.